monero-outreach-docs/translations/archive/wallet_primer_fr.md

```diff
@@ -0,0 +1,90 @@
+
+Allez sur le site web moneroaddress.org dans un navigateur, chargez la page, ensuite coupez votre connexion internet afin de générer la clé. Entrez un texte qui sera très aléatoire (disons, plus de 50 caractères imprédictibles) ou laissez la ligne vide pour utiliser une suite aléatoire généré par ordinateur, et cliquez sur « GENERATE WALLET » (Générer le portefeuille). Ceci vous fournira une clé de dépense privée, une clé d’audit privée, et une clé publique (appelée adresse publique sur ce site web). Sinon, pour une sécurité accrue, le logiciel pour le site web peut être téléchargé en fichier HTML et démarré sur un ordinateur déconnecté, ou le code source peut être téléchargé depuis le Github github.com/moneromooo-monero/monero-wallet-generator/.
+
+Pour la génération d’un portefeuille matériel, vous pouvez utiliser un Ledger ou un Trezor. Les étapes pour la génération de portefeuille avec Ledger sont fournies ci-dessous.
+
+Télécharger Ledger Live depuis [shop.ledger.com/ledger-live](https://www.ledger.com/ledger-live/). Ensuite depuis le Ledger Live, télécharger sur votre Ledger l’application pour Monero. Avec le portefeuille officiel de Monero, utilisez l’application Monero de Ledger pour créer un portefeuille basé sur vos 24 mots mnémoniques, selon la représentation 2. Le portefeuille officiel de Monero a une option pour le Ledger afin de vous assister dans ce processus. Cette méthode de génération de clé a le désavantage de rendre la clé de dépense privé difficile à extraire, mais ce désavantage est compensé par la possibilité d’utiliser le Ledger pour voir les transactions entrantes.
+
+### _Sécurité lors de la génération de portefeuille_
+Des précautions particulières doivent être prises lors de la création de vos clés avec les méthodes décrites ci-dessus. Une monnaie cryptographique privée peut être volée, car c’est facile à dépenser, est de nature immatériel, intraçable, et ne requiert aucune présence physique pour être véhiculée. Soyez prudent, sinon quelqu’un prendra vos Monero. Toutefois la raison doit être utilisée. Tenez compte du fait que si vous avez peu d’argent liquide à dépenser, vous le porterez dans un portefeuille classique, alors que les gros bénéfices d’une entreprise sont transportés dans des véhicules blindés. Vous devez considérer le niveau de sécurité pour votre portefeuille papier de Monero de la même manière, et mesurer l’effort mis dans la sécurité en proportion de vos avoirs. 
```

Comment:
1. [ car c’est facile à dépenser] ==> [ car **elle est** facile à dépenser ; clareté, consistance avec énumération ]
2. [Toutefois la raison doit être utilisée : Tenez compte] ==> [ **Nous vous invitons toutefois à procéder avec discernement: tenez compte**  ; clareté de la phrase]

Reply:
For 2 indeed, a much better translation. I was too much reluctant to use a long sentence so that I stayed too close from the english literal translation...